libsrc/Itk2DicomConverter.cpp

```diff
@@ -285,6 +291,7 @@ namespace dcmqi {
 
         Uint16 segmentNumber;
         CHECK_COND(segdoc->addSegment(segment, segmentNumber /* returns logical segment number */));
+        segNum2Label.insert(make_pair(segmentNumber, label));
```

Comment:
Here, different values of `segmentNumber` can map to the same value of `label`, which may not be unique across input segmentation files. Is this ok?

Reply:
Yes, looking further, this will definitely be a problem, since this map is later used for renumbering segments!

Reply:
Maybe an easy thing to do is to check if label value is present in the map prior to inserting it, and if it is indeed present, reset `sortByLabelID` to `false`, since sorting becomes impossible?

Reply:
I assumed that labels are unique in the input. If this assumption does not hold, then indeed the sorting does not make sense (since the idea to use the original labels as Segment Numbers cannot work then if two labels use the same number)